scripts/core/spellcheck/spellcheck.ts

```diff
@@ -389,6 +389,16 @@ function SpellcheckService($q, api, dictionaries, $rootScope, $location, _, pref
         getItemIgnored()[word] = 1;
     }
 
+    /**
+     * Return list of ignored words
+     *
+     * @return {Array} list of words
+     */
+    this.getIgnoredWords = getIgnoredWords;
```

Comment:
Why not inline the function?

```
this.getIgnoredWords = getIgnoredWords(): Array<string> {
    return Object.keys(getItemIgnored());
}
```

Reply:
I just did like the other methods for consistency (e.g. `ignoreWord` above), as I'm usually not working with client I'm not sure of coding style :). I'll change that :+1: